pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -248,6 +248,11 @@ type Suspension struct {
 	// Note: Can not co-exist with Dispatching which is used to suspend all.
 	// +optional
 	DispatchingOnClusters *SuspendClusters `json:"dispatchingOnClusters,omitempty"`
+
+	// Scheduling controls whether scheduling should be suspended.
+	// nil means not suspend, no default value, only accepts 'true'.
+	// Karmada scheduler will pause scheduling when value is true and resume scheduling when it's nil.
+	Scheduling *bool `json:"scheduling,omitempty"`
```

Comment:
Can you explain a little bit why `PropagationPolicy` needs this?

Reply:
The `Suspension` is common struct which indicates a suspension semantics, so based on the existing `Suspension`, I added scheduling suspension other than dispatcher. If you think it is inappropriate, I can add a new suspension structure just for scheduling, but this seems a bit redundant.

Reply:
Sorry, I didn't make it clear. 
In my understanding, what you need is introducing scheduling suspension to ResourceBinding, my question is do you need to do that to PropgationPolicy as well?

PS:
I know both ResourceBinding and PropgationPolicy share the same typed structure (`type Suspension struct`). They may no longer be able to share the existing structure if we don't want to introduce scheduling suspension to PropgationPolicy.

Reply:
Yeah you are right, currenly I don't need the filed in PropgationPolicy, unless we just want to debug this feature easily.

Reply:
Yeah you are 
right, has modified: )